Formula/lv2.rb

```diff
@@ -14,8 +13,11 @@ class Lv2 < Formula
   end
 
   def install
-    system "./waf", "configure", "--prefix=#{prefix}", "--no-plugins"
+    system "./waf", "configure", "--prefix=#{prefix}", "--lv2dir=#{lib}/lv2/"
     system "./waf", "build"
     system "./waf", "install"
   end
+
+  test do
```

Comment:
I'm a little confused. Does `lv2` install something the user interacts/develops with (e.g., a shared/static library), or does it just install header files? If the former, the test should be simple C file the just gets compiled to ensure linkage. If the latter, it should probably be integrated into packages that depend on it as a `resource` block, rather than being a separate formula.